### ISSUE TITLE
Drops XP// Place spawn bug

### ISSUE THREAD

**Issue by Deernose** [NightlySide/SpawnerChangerGUI#4]
when I break the spawn it drops, but together comes xp// when I put the spawn it turns into spawn pig

**Comment by NightlySide** [comment]
Working on it ;)

**Comment by NightlySide** [comment]
For the XP you can set it in the config file btw

**Comment by Deernose** [comment]
but as I drop xp and spawner I can put it and make infinite xp, I want it to only drop xp when the spawn does not drop, this was a bug, right ?!

**Comment by Deernose** [comment]
tnx you for everything, you are fucking awesome

**Comment by NightlySide** [comment]
Yup that's definitely something that is not wanted ... Right now I'm not near my pc but I'll work on it ASAP ;)

**Comment by Deernose** [comment]
just out of curiosity do you intend to make the spawner gui for 1.13?
